### DISCUSSION TITLE
Github UI

### DISCUSSION THREAD

**Discussion by imrany** [community/community#58398] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

**I prefer the previous github's user interface!!!**

**Comment by radar-blip** [comment, marked as answer]
Not sure if this is what you're looking for, but you can disable this recent update under `Feature Preview` (in your menu).

![globalnavupdate](https://github.com/community/community/assets/39196767/ca822f31-17c0-49eb-8baa-e0a5eb15dc32)

**Reply by imrany** [reply]
Exactly... Thanks, it helped
